src/main/java/uk/ac/cam/cl/dtg/isaac/api/managers/UserAttemptManager.java

```diff
@@ -78,7 +82,15 @@ private void augmentContentSummaryWithAttemptInformation(
                 }
             }
         }
-        contentSummary.setCorrect(questionAnsweredCorrectly);
+        if (attempted) {
+            if (questionAnsweredCorrectly) {
+                contentSummary.setState(ALL_CORRECT);
+            } else {
+                contentSummary.setState(IN_PROGRESS);
+            }
+        } else {
+            contentSummary.setState(NOT_ATTEMPTED);
```

Comment:
I think these would be better scoped, especially since we have another Enum with some of the same item names and it's best to be clear which is which.
`CompletionState.ALL_CORRECT`, etc, and change the import to remove the `.*` on the end.